.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
```

Comment:
- Inconsistent variable naming: `RELEASE_NOTES` is used to store the updated content in the beginning, but later it is used to store the original content with modifications. This can lead to confusion, consider using more descriptive variable names for clarity.